src/actions/user.js

```diff
@@ -10,46 +9,59 @@ const noSuchKey = data => (
   typeof data === 'string' && data.includes('NoSuchKey')
 );
 
-export const SET_USER_SNIPPETS = 'SET_USER_SNIPPETS';
-export const UPDATE_USER_SNIPPETS_STARTED = 'UPDATE_USER_SNIPPETS_STARTED';
-export const UPDATE_USER_SNIPPETS_SUCCEEDED = 'UPDATE_USER_SNIPPETS_SUCCEEDED';
-export const UPDATE_USER_SNIPPETS_FAILED = 'UPDATE_USER_SNIPPETS_FAILED';
-export const UPDATE_USER_SNIPPETS = 'UPDATE_USER_SNIPPETS';
 export const ADD_ORG = 'ADD_ORG';
-export const SWITCH_ORG = 'SWITCH_ORG';
-export const SAVE_USERNAME = 'SAVE_USERNAME';
-export const SAVE_ACCESS_TOKEN = 'SAVE_ACCESS_TOKEN';
+export const ADD_ORGANIZATIONS = 'ADD_ORGANIZATIONS';
```

Comment:
`ADD_ORGANIZATIONS` is a batch version of `ADD_ORG` so that we don't need to make multiple dispatch calls.